NEWS.md

```diff
@@ -1,44 +1,30 @@
 # rsconnect (development version)
 
-* `deployDoc()` and `deployApp()` now support deploying static content to Posit
-  Cloud. Static RMarkdown and Quarto content can be rendered server-side.
-
-* When recording details about deployments to Posit Cloud, appId now represents
-  the content id (as seen in URLs of the format 
-  `https://posit.cloud/content/{id}`) instead of the application id.
-
-* A `version` field has been added to deployment DCF files to facilitate file
-  format changes. Its value for this release is `1`.
-
-* `deployApp()` now gives an actionable error if you attempt to set
-  visiblity of an app deployed to posit.cloud (#838).
+## New features
 
-* `deploySite()` now supports quarto websites (#813).
+* `deployApp()` and `deployDoc()` now support deploying static content to Posit
```

Comment:
This is not a public feature. https://posit.cloud/learn/guide#publish-outputs

Reply:
Fine by me, and is a record of when support was introduced in the client, even if the server support is still in-flight.